pkg/datapath/option/option.go

```diff
@@ -22,4 +22,7 @@ const (
 
 	// DatapathModeIpvlan specifies ipvlan datapath mode.
 	DatapathModeIpvlan = "ipvlan"
+
+	// DatapathModeLB specifies lb-only datapath mode.
+	DatapathModeLB = "lb"
```

Comment:
Maybe make this: `DatapathModeLbOnly = "lb-only"`? Makes code more greppable/readable I think.